app/Http/Controllers/Admin/MonsterCrudController.php

```diff
@@ -16,6 +16,8 @@ class MonsterCrudController extends CrudController
     use \Backpack\CRUD\app\Http\Controllers\Operations\FetchOperation;
     use \Backpack\Pro\Http\Controllers\Operations\DropzoneOperation { dropzoneUpload as traitDropzone; }
     use \App\Http\Controllers\Admin\Operations\SMSOperation; //Custom Form Operation Example
+    use \Backpack\ActivityLog\Http\Controllers\Operations\ShowModelActivityLogsOperation;
```

Comment:
I am just wondering .. if the namespace is already `\Backpack\ActivityLog\` we could probably avoid a name so verbose. I I was thinking in something like `ModelActivityOperation` and `EntryActivityOperation` ?

Reply:
Do we also always use `...\Http\Controllers\Operations\...`?
Is this part required?

Reply:
That's probably better for consistency accross operations in all packages.

Reply:
Ok, so should we rename;
`ShowModelActivityLogsOperation`, to;
`ModelActivityOperation`

Reply:
> That's probably better for consistency accross operations in all packages.

Agreed!

> Ok, so should we rename to `ModelActivityOperation`

Agreed! Good catch @pxpm and good name suggestion! MUCH better!